src/App.tsx

```diff
@@ -73,6 +75,13 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const flowTheme =
+		(urlParams.get('theme') as ThemeOptions) ||
+		(process.env.DESCOPE_THEME as ThemeOptions) ||
+		('light' as ThemeOptions);
```

Comment:
FYI , We don't need to cast each var, we can cast them all together, it will be more readable and less repetitive